packages/web-app-files/src/store/actions.ts

```diff
@@ -522,6 +523,40 @@ export default {
     if (preview) {
       commit('UPDATE_RESOURCE_FIELD', { id: resource.id, field: type, value: preview })
     }
+  },
+
+  loadAncestorMetaData({ commit, state }, { path, space, client }) {
+    const ancestorMetaData = {}
+    const promises = []
+    const parentPaths = getParentPaths(path, true)
+
+    for (const parentPath of parentPaths) {
+      const cachedData = state.ancestorMetaData[parentPath] ?? null
+      if (cachedData?.spaceId === space.id) {
+        ancestorMetaData[parentPath] = cachedData
+        continue
+      }
+
+      promises.push(
+        client
+          .listFiles(
+            space,
+            { path: parentPath },
+            { depth: 0, davProperties: [DavProperty.FileId, DavProperty.ShareTypes] }
```

Comment:
```suggestion
            { depth: 0, davProperties: [DavProperty.FileId, DavProperty.FileParent, DavProperty.ShareTypes] }
```

would make sense to grab the parent file id as well - or in a subsequent PR that adds the parent file id to the breadcrumbs.